src/plugins/irc/irc-server.c

```diff
@@ -63,6 +63,7 @@
 #include "irc-raw.h"
 #include "irc-redirect.h"
 #include "irc-sasl.h"
+#include "../../core/wee-hook.h"
```

Comment:
Weechat plugins (in this case "irc") shouldn't directly interact with the core, that's what the plugin API is for.